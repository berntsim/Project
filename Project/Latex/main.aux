\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{PhysRevLett.47.1400}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Theory}{2}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}On-lattice DLA}{2}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Off-lattice DLA}{2}{subsection.2.2}}
\citation{Kuijpers2014841}
\@writefile{toc}{\contentsline {section}{\numberline {3}Method}{3}{section.3}}
\bibstyle{acm}
\bibdata{bibliography.bib}
\bibcite{Kuijpers2014841}{1}
\bibcite{PhysRevLett.47.1400}{2}
\@writefile{toc}{\contentsline {section}{\numberline {A}References}{4}{appendix.A}}
