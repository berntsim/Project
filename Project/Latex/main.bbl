\begin{thebibliography}{1}

\bibitem{PhysRevLett.47.1400}
T.~A. Witten and L.~M. Sander.
\newblock Diffusion-limited aggregation, a kinetic critical phenomenon.
\newblock {\em Phys. Rev. Lett.}, 47:1400--1403, Nov 1981.

\bibitem{eden1961}
Murray Eden.
\newblock A two-dimensional growth process.
\newblock In {\em Proceedings of the Fourth Berkeley Symposium on Mathematical
  Statistics and Probability, Volume 4: Contributions to Biology and Problems
  of Medicine}, pages 223--239, Berkeley, Calif., 1961. University of
  California Press.

\bibitem{PhysRevLett.51.1119}
Paul Meakin.
\newblock Formation of fractal clusters and networks by irreversible
  diffusion-limited aggregation.
\newblock {\em Phys. Rev. Lett.}, 51:1119--1122, Sep 1983.

\bibitem{PhysRevLett.52.1669}
Tam\'as Vicsek and Fereydoon Family.
\newblock Dynamic scaling for aggregation of clusters.
\newblock {\em Phys. Rev. Lett.}, 52:1669--1672, May 1984.

\bibitem{acpd-15-2487-2015}
B.~V. Scarnato, S.~China, K.~Nielsen, and C.~Mazzoleni.
\newblock Perturbations of the optical properties of mineral dust particles by
  mixing with black carbon: a numerical simulation study.
\newblock {\em Atmospheric Chemistry and Physics Discussions},
  15(2):2487--2533, 2015.

\bibitem{Kuijpers2014841}
Kasper~R. Kuijpers, Lilian de~Mart√≠n, and J.~Ruud van Ommen.
\newblock Optimizing off-lattice diffusion-limited aggregation.
\newblock {\em Computer Physics Communications}, 185(3):841 -- 846, 2014.

\end{thebibliography}
