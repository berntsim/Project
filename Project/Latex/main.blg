This is BibTeX, Version 0.99d (TeX Live 2015)
Capacity: max_strings=35307, hash_size=35307, hash_prime=30011
The top-level auxiliary file: main.aux
The style file: acm.bst
Database file #1: bibliography.bib.bib
You've used 2 entries,
            2253 wiz_defined-function locations,
            545 strings with 4263 characters,
and the built_in function-call counts, 725 in all, are:
= -- 71
> -- 29
< -- 0
+ -- 12
- -- 10
* -- 67
:= -- 122
add.period$ -- 6
call.type$ -- 2
change.case$ -- 11
chr.to.int$ -- 0
cite$ -- 2
duplicate$ -- 18
empty$ -- 59
format.name$ -- 10
if$ -- 146
int.to.chr$ -- 0
int.to.str$ -- 2
missing$ -- 2
newline$ -- 13
num.names$ -- 4
pop$ -- 7
preamble$ -- 1
purify$ -- 9
quote$ -- 0
skip$ -- 10
stack$ -- 0
substring$ -- 68
swap$ -- 4
text.length$ -- 0
text.prefix$ -- 0
top$ -- 0
type$ -- 8
warning$ -- 0
while$ -- 7
width$ -- 3
write$ -- 22
