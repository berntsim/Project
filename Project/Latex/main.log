This is pdfTeX, Version 3.14159265-2.6-1.40.16 (TeX Live 2015) (preloaded format=pdflatex 2015.8.30)  10 OCT 2015 12:54
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**main.tex
(./main.tex
LaTeX2e <2015/01/01>
Babel <3.9l> and hyphenation patterns for 79 languages loaded.
(./include.tex (/usr/local/texlive/2015/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/local/texlive/2015/texmf-dist/tex/latex/base/size12.clo
File: size12.clo 2014/09/29 v1.4h Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2014/10/28 v1.0g Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/local/texlive/2015/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2014/10/28 v1.0p Standard LaTeX Graphics (DPC,SPQR)

(/usr/local/texlive/2015/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 1999/03/16 v1.09 sin cos tan (DPC)
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/latexconfig/graphics.cfg
File: graphics.cfg 2010/04/23 v1.9 graphics configuration of TeX Live
)
Package graphics Info: Driver file: pdftex.def on input line 94.

(/usr/local/texlive/2015/texmf-dist/tex/latex/pdftex-def/pdftex.def
File: pdftex.def 2011/05/27 v0.06d Graphics/color for pdfTeX

(/usr/local/texlive/2015/texmf-dist/tex/generic/oberdiek/infwarerr.sty
Package: infwarerr 2010/04/08 v1.3 Providing info/warning/error messages (HO)
)
(/usr/local/texlive/2015/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
Package: ltxcmds 2011/11/09 v1.22 LaTeX kernel commands for general use (HO)
)
\Gread@gobject=\count87
))
\Gin@req@height=\dimen103
\Gin@req@width=\dimen104
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/base/inputenc.sty
Package: inputenc 2015/03/17 v1.2c Input encoding file
\inpenc@prehook=\toks15
\inpenc@posthook=\toks16

(/usr/local/texlive/2015/texmf-dist/tex/latex/ucs/utf8x.def
File: utf8x.def 2004/10/17 UCS: Input encoding UTF-8
))
(/usr/local/texlive/2015/texmf-dist/tex/latex/ucs/ucs.sty
Package: ucs 2013/05/11 v2.2 UCS: Unicode input support

(/usr/local/texlive/2015/texmf-dist/tex/latex/ucs/data/uni-global.def
File: uni-global.def 2013/05/13 UCS: Unicode global data
)
\uc@secondtry=\count88
\uc@combtoks=\toks17
\uc@combtoksb=\toks18
\uc@temptokena=\toks19
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/graphics/color.sty
Package: color 2014/10/28 v1.1a Standard LaTeX Color (DPC)

(/usr/local/texlive/2015/texmf-dist/tex/latex/latexconfig/color.cfg
File: color.cfg 2007/01/18 v1.5 color configuration of teTeX/TeXLive
)
Package color Info: Driver file: pdftex.def on input line 142.
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/multirow/multirow.sty
\bigstrutjot=\dimen105
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/wrapfig/wrapfig.sty
\wrapoverhang=\dimen106
\WF@size=\dimen107
\c@WF@wrappedlines=\count89
\WF@box=\box26
\WF@everypar=\toks20
Package: wrapfig 2003/01/31  v 3.6
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/caption/subcaption.sty
Package: subcaption 2013/02/03 v1.1-62 Sub-captions (AR)

(/usr/local/texlive/2015/texmf-dist/tex/latex/caption/caption.sty
Package: caption 2013/05/02 v3.3-89 Customizing captions (AR)

(/usr/local/texlive/2015/texmf-dist/tex/latex/caption/caption3.sty
Package: caption3 2013/05/02 v1.6-88 caption3 kernel (AR)
Package caption3 Info: TeX engine: e-TeX on input line 57.
\captionmargin=\dimen108
\captionmargin@=\dimen109
\captionwidth=\dimen110
\caption@tempdima=\dimen111
\caption@indent=\dimen112
\caption@parindent=\dimen113
\caption@hangindent=\dimen114
)
\c@ContinuedFloat=\count90
Package caption Info: wrapfig package is loaded.
)
\c@subfigure=\count91
\c@subtable=\count92
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2012/11/06 v6.83m Hypertext links for LaTeX

(/usr/local/texlive/2015/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty
Package: hobsub-hyperref 2012/05/28 v1.13 Bundle oberdiek, subset hyperref (HO)


(/usr/local/texlive/2015/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty
Package: hobsub-generic 2012/05/28 v1.13 Bundle oberdiek, subset generic (HO)
Package: hobsub 2012/05/28 v1.13 Construct package bundles (HO)
Package hobsub Info: Skipping package `infwarerr' (already loaded).
Package hobsub Info: Skipping package `ltxcmds' (already loaded).
Package: ifluatex 2010/03/01 v1.3 Provides the ifluatex switch (HO)
Package ifluatex Info: LuaTeX not detected.
Package: ifvtex 2010/03/01 v1.5 Detect VTeX and its facilities (HO)
Package ifvtex Info: VTeX not detected.
Package: intcalc 2007/09/27 v1.1 Expandable calculations with integers (HO)
Package: ifpdf 2011/01/30 v2.3 Provides the ifpdf switch (HO)
Package ifpdf Info: pdfTeX in PDF mode is detected.
Package: etexcmds 2011/02/16 v1.5 Avoid name clashes with e-TeX commands (HO)
Package etexcmds Info: Could not find \expanded.
(etexcmds)             That can mean that you are not using pdfTeX 1.50 or
(etexcmds)             that some package has redefined \expanded.
(etexcmds)             In the latter case, load this package earlier.
Package: kvsetkeys 2012/04/25 v1.16 Key value parser (HO)
Package: kvdefinekeys 2011/04/07 v1.3 Define keys (HO)
Package: pdftexcmds 2011/11/29 v0.20 Utility functions of pdfTeX for LuaTeX (HO
)
Package pdftexcmds Info: LuaTeX not detected.
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
Package: pdfescape 2011/11/25 v1.13 Implements pdfTeX's escape features (HO)
Package: bigintcalc 2012/04/08 v1.3 Expandable calculations on big integers (HO
)
Package: bitset 2011/01/30 v1.1 Handle bit-vector datatype (HO)
Package: uniquecounter 2011/01/30 v1.2 Provide unlimited unique counter (HO)
)
Package hobsub Info: Skipping package `hobsub' (already loaded).
Package: letltxmacro 2010/09/02 v1.4 Let assignment for LaTeX macros (HO)
Package: hopatch 2012/05/28 v1.2 Wrapper for package hooks (HO)
Package: xcolor-patch 2011/01/30 xcolor patch
Package: atveryend 2011/06/30 v1.8 Hooks at the very end of document (HO)
Package atveryend Info: \enddocument detected (standard20110627).
Package: atbegshi 2011/10/05 v1.16 At begin shipout hook (HO)
Package: refcount 2011/10/16 v3.4 Data extraction from label references (HO)
Package: hycolor 2011/01/30 v1.7 Color options for hyperref/bookmark (HO)
)
(/usr/local/texlive/2015/texmf-dist/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2010/09/12 v0.6 Provides ifxetex conditional
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/oberdiek/auxhook.sty
Package: auxhook 2011/03/04 v1.3 Hooks for auxiliary files (HO)
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2011/06/30 v3.11 Key value format for package options (HO)
)
\@linkdim=\dimen115
\Hy@linkcounter=\count93
\Hy@pagecounter=\count94

(/usr/local/texlive/2015/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2012/11/06 v6.83m Hyperref: PDFDocEncoding definition (HO)
)
\Hy@SavedSpaceFactor=\count95

(/usr/local/texlive/2015/texmf-dist/tex/latex/latexconfig/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
Package hyperref Info: Hyper figures OFF on input line 4443.
Package hyperref Info: Link nesting OFF on input line 4448.
Package hyperref Info: Hyper index ON on input line 4451.
Package hyperref Info: Plain pages OFF on input line 4458.
Package hyperref Info: Backreferencing OFF on input line 4463.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4688.
\c@Hy@tempcnt=\count96

(/usr/local/texlive/2015/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip10
Package: url 2013/09/16  ver 3.4  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 5041.
\XeTeXLinkMargin=\dimen116
\Fld@menulength=\count97
\Field@Width=\dimen117
\Fld@charsize=\dimen118
Package hyperref Info: Hyper figures OFF on input line 6295.
Package hyperref Info: Link nesting OFF on input line 6300.
Package hyperref Info: Hyper index ON on input line 6303.
Package hyperref Info: backreferencing OFF on input line 6310.
Package hyperref Info: Link coloring OFF on input line 6315.
Package hyperref Info: Link coloring with OCG OFF on input line 6320.
Package hyperref Info: PDF/A mode OFF on input line 6325.
LaTeX Info: Redefining \ref on input line 6365.
LaTeX Info: Redefining \pageref on input line 6369.
\Hy@abspage=\count98
\c@Item=\count99
\c@Hfootnote=\count100
)

Package hyperref Message: Driver (autodetected): hpdftex.

(/usr/local/texlive/2015/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2012/11/06 v6.83m Hyperref driver for pdfTeX
\Fld@listcount=\count101
\c@bookmark@seq@number=\count102

(/usr/local/texlive/2015/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty
Package: rerunfilecheck 2011/04/15 v1.7 Rerun checks for auxiliary files (HO)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
82.
)
\Hy@SectionHShift=\skip43
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/amsmath/amsmath.sty
Package: amsmath 2013/01/14 v2.14 AMS math features
\@mathmargin=\skip44

For additional information on amsmath, use the `?' option.
(/usr/local/texlive/2015/texmf-dist/tex/latex/amsmath/amstext.sty
Package: amstext 2000/06/29 v2.01

(/usr/local/texlive/2015/texmf-dist/tex/latex/amsmath/amsgen.sty
File: amsgen.sty 1999/11/30 v2.0
\@emptytoks=\toks21
\ex@=\dimen119
))
(/usr/local/texlive/2015/texmf-dist/tex/latex/amsmath/amsbsy.sty
Package: amsbsy 1999/11/29 v1.2d
\pmbraise@=\dimen120
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/amsmath/amsopn.sty
Package: amsopn 1999/12/14 v2.01 operator names
)
\inf@bad=\count103
LaTeX Info: Redefining \frac on input line 210.
\uproot@=\count104
\leftroot@=\count105
LaTeX Info: Redefining \overline on input line 306.
\classnum@=\count106
\DOTSCASE@=\count107
LaTeX Info: Redefining \ldots on input line 378.
LaTeX Info: Redefining \dots on input line 381.
LaTeX Info: Redefining \cdots on input line 466.
\Mathstrutbox@=\box27
\strutbox@=\box28
\big@size=\dimen121
LaTeX Font Info:    Redeclaring font encoding OML on input line 566.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 567.
\macc@depth=\count108
\c@MaxMatrixCols=\count109
\dotsspace@=\muskip11
\c@parentequation=\count110
\dspbrk@lvl=\count111
\tag@help=\toks22
\row@=\count112
\column@=\count113
\maxfields@=\count114
\andhelp@=\toks23
\eqnshift@=\dimen122
\alignsep@=\dimen123
\tagshift@=\dimen124
\tagwidth@=\dimen125
\totwidth@=\dimen126
\lineht@=\dimen127
\@envbody=\toks24
\multlinegap=\skip45
\multlinetaggap=\skip46
\mathdisplay@stack=\toks25
LaTeX Info: Redefining \[ on input line 2665.
LaTeX Info: Redefining \] on input line 2666.
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/amsfonts/amssymb.sty
Package: amssymb 2013/01/14 v3.01 AMS font symbols

(/usr/local/texlive/2015/texmf-dist/tex/latex/amsfonts/amsfonts.sty
Package: amsfonts 2013/01/14 v3.01 Basic AMSFonts support
\symAMSa=\mathgroup4
\symAMSb=\mathgroup5
LaTeX Font Info:    Overwriting math alphabet `\mathfrak' in version `bold'
(Font)                  U/euf/m/n --> U/euf/b/n on input line 106.
))
Package hyperref Info: Option `colorlinks' set `true' on input line 23.

(/usr/local/texlive/2015/texmf-dist/tex/latex/titlesec/titlesec.sty
Package: titlesec 2011/12/15 v2.10.0 Sectioning titles
\ttl@box=\box29
\beforetitleunit=\skip47
\aftertitleunit=\skip48
\ttl@plus=\dimen128
\ttl@minus=\dimen129
\ttl@toksa=\toks26
\titlewidth=\dimen130
\titlewidthlast=\dimen131
\titlewidthfirst=\dimen132
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/tools/tabularx.sty
Package: tabularx 2014/10/28 v2.10 `tabularx' package (DPC)

(/usr/local/texlive/2015/texmf-dist/tex/latex/tools/array.sty
Package: array 2014/10/28 v2.4c Tabular extension package (FMi)
\col@sep=\dimen133
\extrarowheight=\dimen134
\NC@list=\toks27
\extratabsurround=\skip49
\backup@length=\skip50
)
\TX@col@width=\dimen135
\TX@old@table=\dimen136
\TX@old@col=\dimen137
\TX@target=\dimen138
\TX@delta=\dimen139
\TX@cols=\count115
\TX@ftn=\toks28
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/parskip/parskip.sty
Package: parskip 2001/04/09 non-zero parskip adjustments
)
(/usr/local/texlive/2015/texmf-dist/tex/latex/tocbibind/tocbibind.sty
Package: tocbibind 2010/10/13 v1.5k extra ToC listings
Package tocbibind Info: The document has section divisions on input line 50.


Package tocbibind Note: Using section or other style headings.

)) (./main.aux)
\openout1 = `main.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 2.
LaTeX Font Info:    ... okay on input line 2.

(/usr/local/texlive/2015/texmf-dist/tex/context/base/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count116
\scratchdimen=\dimen140
\scratchbox=\box30
\nofMPsegments=\count117
\nofMParguments=\count118
\everyMPshowfont=\toks29
\MPscratchCnt=\count119
\MPscratchDim=\dimen141
\MPnumerator=\count120
\makeMPintoPDFobject=\count121
\everyMPtoPDFconversion=\toks30
) (/usr/local/texlive/2015/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
Package: epstopdf-base 2010/02/09 v2.5 Base part for package epstopdf

(/usr/local/texlive/2015/texmf-dist/tex/latex/oberdiek/grfext.sty
Package: grfext 2010/08/19 v1.1 Manage graphics extensions (HO)
)
Package grfext Info: Graphics extension search list:
(grfext)             [.png,.pdf,.jpg,.mps,.jpeg,.jbig2,.jb2,.PNG,.PDF,.JPG,.JPE
G,.JBIG2,.JB2,.eps]
(grfext)             \AppendGraphicsExtensions on input line 452.

(/usr/local/texlive/2015/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
))
(/usr/local/texlive/2015/texmf-dist/tex/latex/ucs/ucsencs.def
File: ucsencs.def 2011/01/21 Fixes to fontencodings LGR, T3
)
Package caption Info: Begin \AtBeginDocument code.
Package caption Info: hyperref package is loaded.
Package caption Info: End \AtBeginDocument code.
\AtBeginShipoutBox=\box31
Package hyperref Info: Link coloring ON on input line 2.

(/usr/local/texlive/2015/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2012/10/27 v2.43 Cross-referencing by name of section

(/usr/local/texlive/2015/texmf-dist/tex/generic/oberdiek/gettitlestring.sty
Package: gettitlestring 2010/12/03 v1.4 Cleanup title references (HO)
)
\c@section@level=\count122
)
LaTeX Info: Redefining \ref on input line 2.
LaTeX Info: Redefining \pageref on input line 2.
LaTeX Info: Redefining \nameref on input line 2.

(./main.out) (./main.out)
\@outlinefile=\write3
\openout3 = `main.out'.

 (./front_page.tex
<fig/NTNU_logo.pdf, id=36, 241.90375pt x 53.19875pt>
File: fig/NTNU_logo.pdf Graphic file (type pdf)
 <use fig/NTNU_logo.pdf>
Package pdftex.def Info: fig/NTNU_logo.pdf used on input line 9.
(pdftex.def)             Requested size: 266.09494pt x 58.51878pt.

Underfull \hbox (badness 10000) in paragraph at lines 11--12

 []

[1

{/usr/local/texlive/2015/texmf-var/fonts/map/pdftex/updmap/pdftex.map} <./fig/N
TNU_logo.pdf>]) (./main.toc
LaTeX Font Info:    Try loading font information for U+msa on input line 3.

(/usr/local/texlive/2015/texmf-dist/tex/latex/amsfonts/umsa.fd
File: umsa.fd 2013/01/14 v3.01 AMS symbols A
)
LaTeX Font Info:    Try loading font information for U+msb on input line 3.

(/usr/local/texlive/2015/texmf-dist/tex/latex/amsfonts/umsb.fd
File: umsb.fd 2013/01/14 v3.01 AMS symbols B
))
\tf@toc=\write4
\openout4 = `main.toc'.

 [1]
(./introduction.tex)pdfTeX warning (ext4): destination with the same identifier
 (name{page.1}) has been already used, duplicate ignored
<to be read again> 
                   \relax 
l.12 \newpage
              [1] (./theory.tex [2]) [3] (./method.tex [4]
<fig/A_sketch.png, id=115, 441.65pt x 441.65pt>
File: fig/A_sketch.png Graphic file (type png)
 <use fig/A_sketch.png>
Package pdftex.def Info: fig/A_sketch.png used on input line 21.
(pdftex.def)             Requested size: 117.00119pt x 116.99625pt.

Underfull \hbox (badness 1931) in paragraph at lines 22--22
[]\OT1/cmr/m/n/10.95 (a) |Il-lus-tra-tion of the
 []


Underfull \hbox (badness 2088) in paragraph at lines 22--22
\OT1/cmr/m/n/10.95 par-ti-cles for sim-plic-ity
 []

<fig/B_array.png, id=117, 441.65pt x 441.65pt>
File: fig/B_array.png Graphic file (type png)
 <use fig/B_array.png>
Package pdftex.def Info: fig/B_array.png used on input line 26.
(pdftex.def)             Requested size: 117.00119pt x 116.99625pt.

<fig/C_array.png, id=118, 437.635pt x 437.635pt>
File: fig/C_array.png Graphic file (type png)
 <use fig/C_array.png>
Package pdftex.def Info: fig/C_array.png used on input line 31.
(pdftex.def)             Requested size: 117.00119pt x 117.0011pt.

Underfull \hbox (badness 2181) in paragraph at lines 32--32
[]\OT1/cmr/m/n/10.95 (c) |Il-lus-tra-tion of the
 []


LaTeX Warning: `h' float specifier changed to `ht'.

[5] [6 <./fig/A_sketch.png> <./fig/B_array.png> <./fig/C_array.png>]) [7]
(./results.tex <fig/on_lattice_400_p_75_g.png, id=143, 602.25pt x 501.875pt>
File: fig/on_lattice_400_p_75_g.png Graphic file (type png)

<use fig/on_lattice_400_p_75_g.png>
Package pdftex.def Info: fig/on_lattice_400_p_75_g.png used on input line 6.
(pdftex.def)             Requested size: 312.00119pt x 260.01158pt.

<fig/1_(1_7248).png, id=148, 1204.5pt x 1003.75pt>
File: fig/1_(1_7248).png Graphic file (type png)
 <use fig/1_(1_7248).png>
Package pdftex.def Info: fig/1_(1_7248).png used on input line 15.
(pdftex.def)             Requested size: 312.00119pt x 260.00394pt.

<fig/loglog(1_7248).png, id=149, 1204.5pt x 1003.75pt>
File: fig/loglog(1_7248).png Graphic file (type png)

<use fig/loglog(1_7248).png>
Package pdftex.def Info: fig/loglog(1_7248).png used on input line 23.
(pdftex.def)             Requested size: 312.00119pt x 260.00394pt.
) [8 <./fig/on_lattice_400_p_75_g.png>] [9 <./fig/1_(1_7248).png>] [10 <./fig/l
oglog(1_7248).png>] (./main.bbl
(/usr/local/texlive/2015/texmf-dist/tex/latex/ucs/data/uni-0.def
File: uni-0.def 2013/05/13 UCS: Unicode data U+0000..U+00FF
))
Package atveryend Info: Empty hook `BeforeClearDocument' on input line 26.
 [11]
Package atveryend Info: Empty hook `AfterLastShipout' on input line 26.

(./main.aux)
Package atveryend Info: Executing hook `AtVeryEndDocument' on input line 26.
Package atveryend Info: Executing hook `AtEndAfterFileList' on input line 26.
Package rerunfilecheck Info: File `main.out' has not changed.
(rerunfilecheck)             Checksum: 07D41FD6565AA5666CC71A1A8772EF4D;410.
Package atveryend Info: Empty hook `AtVeryVeryEnd' on input line 26.
 ) 
Here is how much of TeX's memory you used:
 8408 strings out of 493089
 124364 string characters out of 6134842
 219147 words of memory out of 5000000
 11716 multiletter control sequences out of 15000+600000
 11978 words of font info for 44 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 44i,11n,40p,1408b,636s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx12.pfb
></usr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx8.pfb><
/usr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmcsc10.pfb><
/usr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmex10.pfb></
usr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi12.pfb></u
sr/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi6.pfb></usr
/local/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi8.pfb></usr/l
ocal/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb></usr/loc
al/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmr12.pfb></usr/local
/texlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmr17.pfb></usr/local/t
exlive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmr8.pfb></usr/local/texl
ive/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy10.pfb></usr/local/texli
ve/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy8.pfb></usr/local/texlive
/2015/texmf-dist/fonts/type1/public/amsfonts/cm/cmti12.pfb>
Output written on main.pdf (13 pages, 635406 bytes).
PDF statistics:
 227 PDF objects out of 1000 (max. 8388607)
 185 compressed objects within 2 object streams
 36 named destinations out of 1000 (max. 500000)
 100 words of extra memory for PDF output out of 10000 (max. 10000000)

