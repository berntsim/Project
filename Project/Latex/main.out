\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Theory}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{Fractal dimension}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{Family-Vicsek scaling}{section.2}% 4
\BOOKMARK [1][-]{section.3}{Method}{}% 5
\BOOKMARK [2][-]{subsection.3.1}{On-lattice DLA}{section.3}% 6
\BOOKMARK [2][-]{subsection.3.2}{Off-lattice DLA}{section.3}% 7
\BOOKMARK [2][-]{subsection.3.3}{Diffusion limited cluster aggregation}{section.3}% 8
\BOOKMARK [1][-]{section.4}{Results}{}% 9
\BOOKMARK [1][-]{appendix.A}{References}{}% 10
