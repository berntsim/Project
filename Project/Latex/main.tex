\input{include.tex}
\begin{document}
\input{front_page.tex}% Forsiden
\setcounter{page}{0}
\pagenumbering{roman}
%\addbibresource{refs.bib}
\tableofcontents
\newpage
\setcounter{page}{1}
\pagenumbering{arabic}
%%\input{preface.tex}
%%\newpage
\input{introduction.tex}
\newpage
\input{theory.tex}
\newpage
\input{method.tex}
\newpage
\input{results.tex}
\appendix
\newpage
\bibliographystyle{unsrt}
%\addcontentsline{toc}{section}{\bibname}
\bibliography{bibliography.bib}
%\nocite{*}
%\printbibliography

\end{document}