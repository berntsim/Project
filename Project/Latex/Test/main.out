\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Theory}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{Fractal dimension}{section.2}% 3
\BOOKMARK [1][-]{section.3}{Method}{}% 4
\BOOKMARK [2][-]{subsection.3.1}{On-lattice DLA}{section.3}% 5
\BOOKMARK [2][-]{subsection.3.2}{Off-lattice DLA}{section.3}% 6
\BOOKMARK [1][-]{section.4}{Results}{}% 7
\BOOKMARK [1][-]{appendix.A}{References}{}% 8
